metrics/density/footprint_data.md

```diff
@@ -0,0 +1,107 @@
+# Footprint data script details
+
+The `footprint_data.sh` script runs a number of identical containers sequentially
+via the docker engine and takes a number of memory related measurements after each
+launch. The script is generally not used in a CI type environment, but is intended
+to be run and analyzed manually.
+
+The script can be configured by setting a number of environment variables.
+
+Details of the configurable variables are listed below, along with a small example
+invocation script.
+
+## Variables
+Environment variables can take effect in two ways.
+
+Some variables affect how the payload is executed. The `RUNTIME` and `PAYLOAD`
+arguments directly affect the payload execution via this line in the script:
```

Comment:
arguments directly affect the payload execution with the following line in the script: